Designer heading and spacing

jcfilben opened the pull request:
Changed the designer heading to be more responsive, the icons below the designer have increasing opacity as they get further down the page. I changes the spacing on the 'what is the designer' section. I also changed the images in the capabilities section to be row-responsive

requesting review from @JoeButler7

ShimiSun commented on src/pages/Designer/Designer.js:
```diff
@@ -25,35 +25,111 @@ const Designer = () => (
-                Designer
-              </Heading>
+                <Box direction="row" margin={{ left: 'xlarge' }} gap="xlarge">
+                  <Camera opacity="0.8" size="large" color="ImagerOrange" />
+                  <Configure opacity="0.9" size="large" color="ThemerOrange" />
```
Icon doesn't have an `opacity` prop.

jcfilben replied:
I wasn't sure how else to change the opacity for the icons

ShimiSun replied:
Did it actually work? do you see errors on the console?

jcfilben replied:
Yeah it worked and no errors on the console

ShimiSun replied:
I'll need to take a look at it later.

ShimiSun commented on src/pages/Designer/Designer.js:
```diff
@@ -25,35 +25,111 @@ const Designer = () => (
-                }}
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
```
Instead of having a super long 'if' statement, try to simplify the code using functional components.
Something like:
xyz ? <Component1 /> : <Component2 />.

ShimiSun commented on src/pages/Designer/Designer.js:
```diff
@@ -25,35 +25,111 @@ const Designer = () => (
+                  Designer
+                </Heading>
+              </Box>
+              <Box height="gradient1" background="gradient" />
```
I think you don't need to use `height` and you can remove the definition for "gradient1" on the theme.
You can just use `pad="small"` instead or other t-shirt size.

ShimiSun commented on src/pages/Designer/Designer.js:
```diff
@@ -25,35 +25,111 @@ const Designer = () => (
+                  size="xxlarge"
+                  color="darkGrey"
+                >
+                  Grommet Designer is a tool used to build expiriences with
+                  Grommet components; then publish and share your ideas with a
+                  simple wysiwg interface
```
This is duplicated, extract it to a string.

jcfilben commented:
Update: I removed the set gradient sizes from the designer page and the home page. I changed the responsive context so that it is not a super long if statement.

I didn't get rid of the `opacity` prop on the icons because I wasn't sure how else to set the opacity on them

ShimiSun commented:
I'll merge this PR, but there is still plenty of empty space that needs to be addressed 
![image](https://user-images.githubusercontent.com/6320236/88440823-587f9480-cdcc-11ea-8eb4-72aa92863e1d.png)
